attrs/utils.go

```diff
@@ -0,0 +1,20 @@
+package attrs
+
+import "strings"
+
+// Return name for a data attribute
+func DataAttr(name string) string {
+	var builder strings.Builder
+	builder.WriteString(DataPrefix)
+	builder.WriteString(name)
+	return builder.String()
+}
+
+// Return Props of data attributes build from given name-value pairs
+func DataAttrs(pairs map[string]string) Props {
```

Comment:
Great idea on also adding a way to add multiple considering these may be less unique that other attributes! What do you think about introducing a reusable type named `KeyValueMap` or something similar, to avoid constantly writing `map[string]string`? It could streamline things for users. However, I'm a bit cautious about adding complexity that might make the library harder for developers to grasp. What do you think? Is the trade-off worth it?